package/Dockerfile

```diff
@@ -1,20 +1,26 @@
-FROM ghcr.io/oracle/oraclelinux:7-slim
+FROM ghcr.io/oracle/oraclelinux:8-slim as build
 ARG ARCH
 ENV ARCH=$ARCH
-RUN yum-config-manager --enable ol7_optional_latest && \
-    yum-config-manager --enable ol7_addons && \
-    yum update -y && \
-    # software collections repo needed for git 2.x on OL7
-    yum-config-manager --add-repo=http://yum.oracle.com/repo/OracleLinux/OL7/SoftwareCollections/x86_64 && \
-    yum install -y rh-git227 curl ca-certificates unzip tar docker-cli nc mkisofs && \
-    yum clean all && \
-    rm -rf /var/cache/yum && \
-    # Set up needed to ensure git 2.27 from rh-git227 is on the path
-    ln /opt/rh/rh-git227/enable /etc/profile.d/git.sh && \
-    source /etc/profile.d/git.sh && \
-    cat /etc/profile.d/git.sh >> /etc/profile && \
-    git version
-COPY bin/fleetcontroller-linux-${ARCH} /usr/bin/fleetcontroller
+
+COPY . /go/src/github.com/rancher/fleet/
+
+RUN microdnf update -y --setopt=install_weak_deps=0 --setopt=tsflags=nodocs && \
+    microdnf install -y golang-1.20.10-1.module+el8.9.0+90068+08775a1f.x86_64 && \
+    microdnf install -y git && \
+    cd /go/src/github.com/rancher/fleet/scripts && \
+    ./build && \
+    ./test
+
+FROM ghcr.io/oracle/oraclelinux:8-slim
+ARG ARCH
+ENV ARCH=$ARCH
+
+RUN microdnf update -y --setopt=install_weak_deps=0 --setopt=tsflags=nodocs && \
```

Comment:
noticed docker-cli and tar are not included explicitly in microdnf calls.  Are they available in the base image or not needed?

Reply:
Note that tar is used by 'kubectl cp' so I wonder if it's needed in certain fleet scenarios?

Reply:
they are in the base image

Reply:
same with curl and ca-certificates